e2e/tests/ssh/ssh.go

```diff
@@ -80,6 +80,8 @@ var _ = DevPodDescribe("devpod ssh test suite", func() {
 			err = f.DevPodUp(devpodUpCtx, tempDir, "--gpg-agent-forwarding")
 			framework.ExpectNoError(err)
 
+			time.Sleep(time.Second * 10)
```

Comment:
Out of curiosity, why are we sleeping here?

Reply:
It was a test, I was suspecting some sort of racing/unready-ness because it was failing a bit randomly

I suspect at this point, it's a matter of the gpg agent not being "ready" for the forwarding, leading to some EOF errors during the forwarding

Will do a retry logic in place of this